codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -353,25 +396,62 @@ class ServerServiceGeneratorV2(
                 }
             }
             """,
-            "InternalFailureGenerics" to internalFailureGenerics.join(", "),
-            "InternalFailureFields" to internalFailureFields.join(", "),
-            "NotSetGenerics" to notSetGenerics.join(", "),
             "NotSetFields" to notSetFields.join(", "),
             "Router" to protocol.routerType(),
             "Protocol" to protocol.markerStruct(),
             *codegenScope,
         )
     }
 
+    private fun missingOperationsError(): Writable = writable {
+        rustTemplate(
+            """
+            ##[derive(Debug)]
+            pub struct MissingOperationsError {
+                service_name: &'static str,
+                operation_names2setter_methods: std::collections::HashMap<&'static str, &'static str>,
+            }
+
+            impl std::fmt::Display for MissingOperationsError {
+                fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+                    write!(
+                        f,
+                        "You must specify a handler for all operations attached to the `{}` service.\n\
+                        We are missing handlers for the following operations:\n",
+                        self.service_name
+                    )?;
+                    for operation_name in self.operation_names2setter_methods.keys() {
+                        writeln!(f, "- {}", operation_name)?;
+                    }
+
+                    writeln!(f, "\nUse the dedicated methods on `PokemonServiceBuilder` to register the missing handlers:")?;
```

Comment:
Missing the template for `PokemonServiceBuilder` here.

Reply:
Great catch 💯 
Fixed it in https://github.com/awslabs/smithy-rs/pull/1954/commits/ef089e6a587ed84874f9268636daff6a036f5423